### ISSUE TITLE
Некорректная работа оценки опасности у /firedoor.

### ISSUE THREAD

**Issue by ZVee** [TauCetiStation/TauCetiClassic#1224]
#### Подробное описание проблемы
Занимаясь совершенно другим вопросом заметил, что файрдоры совсем неверно оценивают окружающую среду - вокруг разгерма, опасная ситуация - файрдору кажется все в полном порядке, но пару турфов левее - соседи во всю бъют тревогу (пните меня если уже репортили это).

#### Что должно было произойти
Исходя из кода - сигналы должны загораться исходя из конкретной работы и ровно в том направлении, в котором находится опасность.
#### Как повторить
Поделать разгермы, понаблюдать за работой.
#### Дополнительная информация:
<details><summary>Картинки:</summary>

![fdoors](https://cloud.githubusercontent.com/assets/8426718/23703258/baf8b0e8-0407-11e7-9a8b-00f9d0bd5146.png)
![fdoors2](https://cloud.githubusercontent.com/assets/8426718/23703269/c3f6c1ee-0407-11e7-9c12-327eebfd1d85.png)

</details>

**Comment by Raschert0** [comment]
Ну, там с тем шлюзами около дверей проблема том, что они, видимо, не хотятя сравнивать атмосферу у турфа с закрытой дверью. А вообще-то они сигнализируют не о разгерме, а о разнице давлений. А какая разница давлений омжет быть, если с обеих сторон вакуум?

**Comment by Dennok** [comment]
Да, огоньки сломали. И да. не могут считіть атмос с тайла двери. Но с вариантом двери надо смотреть руками. Может оно обновит когда откроеш внутреннюю дверь.
